pkgs/top-level/all-packages.nix

```diff
@@ -13835,7 +13835,12 @@ let
 
   jags = callPackage ../applications/science/math/jags { };
 
-  liblapack = callPackage ../development/libraries/science/math/liblapack { };
+  liblapack = callPackage ../development/libraries/science/math/liblapack { 
+    atlas = null;
+  };
```

Comment:
I honestly couldn't say. Perhaps I could make an `liblapackWithoutAtlas` expression and make the one with atlas the default.